documentation/book/proc-using-openshift-builds-create-image.adoc

```diff
@@ -4,17 +4,17 @@
 // assembly-deployment-configuration-kafka-connect-s2i.adoc
 
 [id='using-openshift-s2i-create-image-{context}']
-= Creating a container image using {OpenShiftName} builds and Source-to-Image
+= Creating a container image using OpenShift builds and Source-to-Image
 
-You can use {OpenShiftName} link:https://docs.openshift.org/3.9/dev_guide/builds/index.html[builds^] and the  link:https://docs.openshift.org/3.9/creating_images/s2i.html#creating-images-s2i[Source-to-Image (S2I)^] framework to create new container images. An OpenShift build takes a builder image with S2I support, together with source code and binaries provided by the user, and uses them to build a new container image. Once built, container images are stored in {OpenShiftName}'s local container image repository and are available for use in deployments.
+You can use OpenShift link:https://docs.openshift.org/3.9/dev_guide/builds/index.html[builds^] and the  link:https://docs.openshift.org/3.9/creating_images/s2i.html#creating-images-s2i[Source-to-Image (S2I)^] framework to create new container images. An OpenShift build takes a builder image with S2I support, together with source code and binaries provided by the user, and uses them to build a new container image. Once built, container images are stored in OpenShift's local container image repository and are available for use in deployments.
```

Comment:
Updated links to 3.11.

```suggestion
You can use OpenShift link:https://docs.okd.io/3.11/dev_guide/builds/index.html[builds^] and the  link:https://docs.okd.io/3.11/creating_images/s2i.html[Source-to-Image (S2I)^] framework to create new container images.
An OpenShift build takes a builder image with S2I support, together with source code and binaries provided by the user, and uses them to build a new container image.
Once built, container images are stored in OpenShift's local container image repository and are available for use in deployments.
```

Reply:
I'll apply this, but I wonder whether we should generally use attributes for all links to external sites. It seems that we've got a load of rotting/out-dated links scattered through the docs. If they were defined in one place it would make it easier to see when a link was out-dated due to a new release of kubernetes or whatever. It would also mean we had a single place to fix it. Not for this PR, obviously, but do with think this would be a worthwhile approach?